packages/fluentui/react-northstar/src/components/Table/Table.tsx

```diff
@@ -51,34 +58,38 @@ export const tableSlotClassNames: TableSlotClassNames = {
   header: `${tableClassName}__header`,
 };
 
-class Table extends UIComponent<WithAsProp<TableProps>> {
-  static displayName = 'Table';
-  static deprecated_className = tableClassName;
-
-  static Cell = TableCell;
-  static Row = TableRow;
-
-  static propTypes = {
-    ...commonPropTypes.createCommon({
-      content: false,
+export type TableStylesProps = Required<Pick<TableProps, 'variables'>>;
```

Comment:
I don't think we need to pick variables, @mnajdova @layershifter ?

Reply:
Yep, it's true.

```suggestion
export type TableStylesProps = never
```

Reply:
Done :)